core-bundle/tests/Translation/TranslatorTest.php

```diff
@@ -56,14 +59,35 @@ public function testForwardsTheMethodCallsToTheDecoratedTranslator(): void
 
         $translator = new Translator($originalTranslator, $framework);
 
-        $this->assertSame('trans', $translator->trans('id', ['param' => 'value'], 'domain', 'en'));
-        $this->assertSame('transChoice', $translator->transChoice('id', 3, ['param' => 'value'], 'domain', 'en'));
+        $this->assertSame('trans', $translator->trans('id', ['param' => 'value'], $domain, 'en'));
+        $this->assertSame('transChoice', $translator->transChoice('id', 3, ['param' => 'value'], $domain, 'en'));
 
         $translator->setLocale('en');
 
         $this->assertSame('en', $translator->getLocale());
     }
 
+    public function decoratedTranslatorDomainProvider()
+    {
+        yield ['domain'];
+
+        yield ['contao_calendar'];
+
+        yield ['contao_comments'];
+
+        yield ['contao_faq'];
+
+        yield ['contao_installation'];
+
+        yield ['contao_listing'];
+
+        yield ['contao_manager'];
+
+        yield ['contao_news'];
+
+        yield ['contao_newsletter'];
```

Comment:
We should probably also test that `contao_default` and `contao_tl_page` are not forwarded.

Reply:
We should add a lot of tests here, but that's not really related? But feel free to add 🙃